### ISSUE TITLE
Error in "UPDATE ... JOIN" statement

### ISSUE THREAD

**Issue by tyru** [joe-re/sql-language-server#11]
https://www.mysqltutorial.org/mysql-update-join/

```sql
UPDATE T1
JOIN T2
ON T1.t2_id = T2.id
SET T1.name = T2.name
WHERE (... some conditions ...);
```

```
sql:Error:Expected ".", "SET", or [ \t\n\r] but "J" found.
```

**Comment by joe-re** [comment]
@tyru 
sorry, right now sql-language-server supports only SELECT statement..
I have a plan to deal with it though..
Thank you.

**Comment by joe-re** [comment]
fixed by #19 
@tyru Can you check it with latest version of language server? Thank you.
